skywalking-collector/skywalking-collector-worker/src/main/java/com/a/eye/skywalking/collector/worker/storage/MergeAnalysisData.java

```diff
@@ -0,0 +1,23 @@
+package com.a.eye.skywalking.collector.worker.storage;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * @author pengys5
+ */
+public class MergeAnalysisData {
+
+    private WindowData<MergeData> windowData = new WindowData(new LinkedHashMap<String, MergeData>());
+
+    public MergeData getElseCreate(String id) {
```

Comment:
I think your meaning is `getOrCreate`? I think your full statement is `get by id, if not exist, create one`, right?